packages/elements/src/slider/__test__/slider.test.js

```diff
@@ -18,7 +18,17 @@ describe('slider/Slider', function () {
     });
     it('DOM structure with markers is correct', async function () {
       const el = await fixture(`
-      <ef-slider>
+      <ef-slider value="100">
+        <ef-slider-marker value="0">0</ef-slider-marker>
+        <ef-slider-marker value="10">10</ef-slider-marker>
+        <ef-slider-marker value="50"></ef-slider-marker>
+        <ef-slider-marker value="100">100</ef-slider-marker>
+      </ef-slider>`);
+      await expect(el).shadowDom.to.equalSnapshot();
+    });
+    it('LightDOM structure with markers is correct', async function () {
+      const el = await fixture(`
+      <ef-slider value="10">
```

Comment:
using the same value as above should be simpler, right?
```suggestion
      <ef-slider value="100">
```